pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -65,6 +63,7 @@ type baseOsMgrContext struct {
 
 	subGlobalConfig          pubsub.Subscription
 	globalConfig             *types.ConfigItemValueMap
+	agentBaseContext         agentbase.Context
```

Comment:
Keep this as the first member in the object as a convention.. Makes it easy to read..